### DISCUSSION TITLE
Where can I download the single file version？

### DISCUSSION THREAD

**Discussion by jianliulin** [pawelsalawa/sqlitestudio#3918] [Q&A]
### Details
Where can I download the single file version？

### SQLiteStudio version
v 3.21

**Comment by pawelsalawa** [comment]
Single file version was available only for line 1.x and 2.x. The 3.x is no longer released as single executable.
Still it is portable (you can download portable distribution or the one with installer), although it will be multiple files grouped within a directory.
